sample-status-updater-application/src/main/java/life/qbic/samplestatusupdater/app/SampleUpdater.java

```diff
@@ -84,7 +94,7 @@ public void execute() {
         } catch (Exception e) {
             exitCode = 1;
         }
-        properties.put("lastSearchDate", new SimpleDateFormat("yyyy-MM-dd").format(new Date()));
+        properties.put("lastSearchDate", timePointBeforeSearch.toString()) ;
```

Comment:
As the code does not exit but only sets the exit value, the `lastSearchDate` is written even if unsuccessful. Is this desired?

Reply:
good catch!

Reply:
Only caught that because of tests in the reporter app 😄